lib/src/main/java/com/shopify/checkoutsheetkit/CheckoutWebViewEventProcessor.kt

```diff
@@ -25,6 +25,7 @@ package com.shopify.checkoutsheetkit
 import android.net.Uri
 import android.os.Handler
 import android.os.Looper
+import com.shopify.checkoutsheetkit.completedevent.CheckoutCompletedEvent
```

Comment:
Might make more sense to group them under `events`

```suggestion
import com.shopify.checkoutsheetkit.events.CheckoutCompletedEvent
```

Reply:
Sounds good